hyq_angle : [0,0,0]
hyq_h : 0.0

hyq_auto : False
